### PR TITLE
Add icon data to README template + Icon folder

### PR DESCRIPTION
**Pull request by xzi**
See also:
https://github.com/PrismLauncher/prismlauncher.org/pull/265

### CODE REVIEW

File: icons/this file only exists to get github to make a folder [hunk 1]
```diff
@@ -0,0 +1 @@
+Rucker#1337 was here
```
Comment (Lutitious) [inline icons/this file only exists to get github to make a folder]:
```suggestion
<!--
SPDX-FileCopyrightText: "Year" "Nickname or Name"
SPDX-License-Identifier: CC0-1.0
-->
Rucker#1337 was here
```
Sticking a blank copyright identifier in this file should make gh actions happy

Comment (xzi) [reply icons/this file only exists to get github to make a folder]: Thank you <3

File: icons/this file only exists to get github to make a folder [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2023 Lilith <Rucker#1337@discord.com>
```
Comment (getchoo) [inline icons/this file only exists to get github to make a folder:1]: a better option here would probably be to make an empty `.gitkeep` file

**Review by getchoo — APPROVED** [review]
lgtm! thanks :)

**Comment by DioEgizio** [comment]
This pr should also change the CI to create zips for icons

**Comment by TheKodeToad** [comment]
Should be done now

However, why do we need a separate folder for icons? They can't be named the same thing anyway...

**Review by DioEgizio — APPROVED** [review]

**Pull request merged**